### DISCUSSION TITLE
Wolf iSM7 and eBus Adapter

### DISCUSSION THREAD

**Discussion by MiniOh** [john30/ebusd#659] [General]
Hello,
I have a WOLF COB-20 heater incl. solar model.
The system is connected to the Wolf Smartset Cloud via iSM7 (eBus).
Is it possible to operate the eBus adapter and the iSM7 module in parallel, or can only one module be attached to the COB and transmit data?
Many thanks in advance.
Olivier

**Comment by john30** [comment]
yes, it is a bus, you can connect another device to it. only have to make sure that there is no address conflict then
